README.md

```diff
@@ -1,39 +1,21 @@
-# Knights of Academia discord bot
-
+# Horace
 <div align='center'>
   <p>
     <img src='https://img.shields.io/discord/382364344731828224.svg?color=7289DA&label=Discord%20Chat&logo=Discord&logoColor=7289DA&style=plastic' alt='Discord members'>
   </p>
 </div>
 
-### Installation guide
-
-#### Perquisites
-
-This project is based on [NodeJS](https://nodejs.org/en/), which is required to run it.
-
-#### Installing Dependencies
-
-This bot uses multiple dependencies, which can be installed by running `npm install` in the root folder after installing NodeJS.
-
-#### Get a Discord API key
-
-Go to [Developers Section](https://discordapp.com/developers) and create an application by following the instructions.
-The API key you have generated by this is inserted into the **config.json** file to connect the bot authentication to your instance of the bot.
-
-### Inviting the bot to your test server
-
-To invite the bot to your development server, visit the following link where you replace **XXXX** with the **client_id** of your bot, which can be found on the site from the previous step: [https://discordapp.com/oauth2/authorize?client_id=XXXXX&scope=bot](https://discordapp.com/oauth2/authorize?client_id=XXXXX&scope=bot).
+### Who is Horace?
 
-### Customizing the configuration
-The file **config.json.example** contains the environment variables that should be set for the bot to work.
-It requires your discord API token, COTW channel, champion role and manager role.
-In addition to this, it requires your Habitica token, to be able to fetch the COTW information from Habitica API.
+As part of his duties overlooking KOA, Sir Horace is available to assist users in the Knights of Academia Discord server in response to a variety of commands.
+Horace is a pillar of the community. With his ever-watchful eye, he's always ready to welcome new recruits and teach them the ways about KOA and how to improve themselves.
+He is more than just another Discord bot, he's an essential helper for a continually growing community full of individuals wishing to become the best version of themselves that they possibly can.
 
-### Powering up the bot
+Want to contribute to Horace? Check out our [Guide for Contributing](CONTRIBUTING.md).
```

Comment:
I realized that local links don't work, and I'll have to go back and edit the CONTRIBUTING file as well, but we'll need a direct link to it, or somehow change the directory.

Reply:
Updated to fix.